TestAssets/TestPackages/dotnet-dependency-tool-invoker/dotnet-dependency-tool-invoker.csproj

```diff
@@ -13,13 +11,9 @@
     <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <SdkNugetVersion Condition=" '$(SdkNugetVersion)' == ''">$(CliVersionPrefix)-*</SdkNugetVersion>
-  </PropertyGroup>
-
   <ItemGroup>
     <PackageReference Include="NuGet.Frameworks" Version="$(NuGetFrameworksPackageVersion)" />
-    <PackageReference Include="Microsoft.DotNet.Cli.Utils" Version="$(SdkNugetVersion)" />
+    <PackageReference Include="Microsoft.DotNet.Cli.Utils" Version="2.0.0" />
```

Comment:
The reason why we make this depend on the CLI.Utils from the CLI is because it tests part of the code there. If we won't have that code anymore, maybe we should delete these tests.

Reply:
From the issue description I thought it was a test helper. If so, let me see if I can use project reference https://github.com/dotnet/cli/pull/2087

Reply:
It is reference quiet a lot and used in project tools tests

Reply:
Does it depend on the types you moved? I don't think we can rely on an older version in source build.

Reply:
Good news, seems every test used `DependencyToolInvokerCommand` (other than one is asserting failure) is disabled due to https://github.com/dotnet/cli/issues/4514 and the issue is closed. I will see if I can remove all the tests depend on it

Reply:
@livarcocc @nguerrera all tests used `DependencyToolInvokerCommand` are disabled. I removed the whole project and the build still success.

I believe this DependencyToolInvokerCommand is a project tool that can invoke another project tool. However, it requires  `ProjectCommandFactory` which is class in cli.util. And we stopped publishing it since 2.0. This should not be a supported scenario anymore

Reply:
Awesome. I haven't re-review the files, but did you delete these disabled tests? We should.